core/templates/dev/head/components/RatingComputationService.js

```diff
@@ -46,6 +46,14 @@ oppia.factory('RatingComputationService', [function() {
 
         return totalValue / totalNumber;
       }
+    },
+    getRelativeChangeInAverageRating: function(newRating, oldRating) {
+      if (!oldRating) {
+        return '+' + parseFloat(newRating).toFixed(2);
```

Comment:
I'm not sure I understand the use of `parseFloat` here. From the tests you've written, it appears the arguments `newRating` and `oldRating` are numbers, right?